src/main/java/org/kuali/kfs/sys/aws/AmazonSecretValidationInstance.java

```diff
@@ -0,0 +1,49 @@
+package org.kuali.kfs.sys.aws;
```

Comment:
interesting question.  I see this batch job as something that will live for a few months and then we remove once we have confidence this works as we hope, or if we decide to go in a different direction.
I thought it would make sense to isolate this code to its own package structure so it could be removed later.  What do you think?

Reply:
We typically use the "org.kuali.kfs" package structure for overlays of base financials code. In my opinion, putting this AWS code in there would cause confusion as to whether it was actually an overlay of some base code, unless we documented things accordingly.

I would recommend putting this code under the "edu.cornell.kfs.sys.aws" package structure instead.

Reply:
oh!!!!!!!, thank you for noticing, I see now, they are in org, not in edu